fitness_check.py

```diff
@@ -34,6 +37,39 @@ def read_data(sections, genes, dna, filename):
                     individual = individual + 1
 
 
+def combine_scores(current_fitness, previous_fitness,
+                   current_error, previous_error):
+    """Combine scores of identical individuals in the previous generation."""
+    matches = 0
+    for i in range(0, len(current_fitness)):
+        for j in range(0, len(previous_fitness)):
+            if current_dna[i] == previous_dna[j]:
+                matches = matches + 1
+                current_weight = 1.0/(current_error[i]**2.0)
+                previous_weight = 1.0/(previous_error[j]**2.0)
+                current_fitness[i] = ((current_weight * current_fitness[i]
+                                       + previous_weight * previous_fitness[j])
+                                      / (current_weight + previous_weight))
+                current_error[i] = (1.0 /
+                                    ((current_weight + previous_weight)**(0.5))
+                                    )
+    print(matches, "Matches found")
+
+
+def write_fitness(fitness, error):
+    """Write fitnessScore.csv with data for the generation."""
+    with open('fitnessScores.csv', "r") as fs:
+        lines = fs.readlines()
+    lines2 = []
+    with open('fitnessScores.csv', "w") as f2:
+        for x in range(0, len(fitness)+2):
+            if x <= 1:
+                lines2.append(f"{lines[x]}")
```

Comment:
```suggestion
                lines2.append(lines[x])
```
Since you just read it from the other file, it should already be a string.